examples/features/interceptor/README.md

```diff
@@ -1,9 +1,9 @@
 # Interceptor
 
 gRPC provides simple APIs to implement and install interceptors on a per
-ClientConn/Server basis. Interceptor intercepts the execution of each RPC call.
-Users can use interceptors to do logging, authentication/authorization, metrics
-collection, and many other functionality that can be shared across RPCs.
+ClientConn/Server basis. An interceptor intercepts the execution of each RPC
+call. Users can use interceptors for logging, authentication/authorization,
```

Comment:
"Users can use" -> "Interceptors can be used"

Reply:
Changed.